src/test/FreeC/Pass/ExportPassTests.hs

```diff
@@ -0,0 +1,117 @@
+-- | This module contains tests for "FreeC.Pass.ExportPass".
+
+module FreeC.Pass.ExportPassTests
+  ( testExportPass
+  )
+where
+
+import           Test.Hspec
+
+import           Data.List                      ( find )
+import           Data.Maybe                     ( fromJust )
+import qualified Data.Set                      as Set
+import qualified Data.Text                     as Text
+
+import qualified FreeC.Backend.Coq.Syntax      as Coq
+import           FreeC.Environment
+import           FreeC.Environment.Entry
+import           FreeC.Environment.ModuleInterface
+import           FreeC.IR.Syntax               as IR
+import           FreeC.Monad.Class.Testable
+import           FreeC.Monad.Converter
+import           FreeC.Pass.ExportPass
+import           FreeC.Pretty
+import           FreeC.Test.Environment
+import           FreeC.Test.Parser
+
+-- | looks up an exported entry in the given module interface
```

Comment:
Comments are full sentences, so they should be capitalized and end with a period.
```suggestion
-- | Looks up an exported entry in the given module interface.
```